### DISCUSSION TITLE
PatchTool 6.58 3/19

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#426] [Announcements]
6.58相容性更新。

技能數值更新至6.58。

請從release下方連結或內文連結下載檔案。以下兩種方法擇一使用即可。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20240319.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20240319.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-4k-smaller-20230131.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.0.1/font-custom-4k-smaller-20230131.rar)：高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)透過重複使用字體字型縮小檔案體積，使用Noto Sans。
- [font-custom-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.2/font-custom-20230111.rar)：舊高畫質字體包，請取代原本的resource/font資料夾。[使用此字體包需更改設定。](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)所有大小字體皆分開繪製，使用Noto Sans和Noto Serif。
- rawexd-zhcn-20240319.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- FFXIVChnTextPatch-20240319.jar：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- backup-20240319.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.2.1.5'>PatchTool 6.58 3/19</a>.</em>

**Comment by belle0976** [comment]
您好，感謝您即時更新新漢化，辛苦了。
這次更新，我遇到無法開啟遊戲的問題，當登入器要開啟遊戲時，會跳出以下視窗，連遊戲畫面都沒開啟就關閉了，不曉得是電腦哪裡我沒更新到的？
1.已確定是更換新漢化才無法開啟遊戲，換回官方原始檔就能正常遊戲了。
2.已更新最新的顯卡軟體，問題依舊。
![螢幕擷取畫面 2024-03-20 190949](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/95540085/0ddedbfb-4447-4e19-8c06-c47d8155900a)
PS︰我是用方法1，直接覆蓋遊戲檔案。
非常感謝您，一直是您漢化包的愛用者 ^_^ ♡

**Reply by GpointChen** [reply]
我目前沒有遇過這個問題，沒辦法復現，還是你先試試看方法二？

**Reply by belle0976** [reply]
好的，我再試試看，謝謝您～

**Reply by belle0976** [reply]
您好，目前使用方法二執行漢化，就可以正常遊戲了，謝謝您～♡

**Comment by tomokiliyo** [comment]
![Snipaste_2024-03-20_21-13-20](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/164188275/dd687ee9-927c-40d6-9c34-c882ba7b096c)
您好，感謝漢化，方才更新後開啟遊戲會跳上面這個錯誤，是使用CSV漢化，不確定如何解決，恢復原語言的話可以正常啟動，感謝您！

**Reply by GpointChen** [reply]
你是用一般的字體包嗎？

**Reply by GpointChen** [reply]
另外，可以確定一下你在更新前是否有先還原漢化。

**Reply by tomokiliyo** [reply]
您好，是用高畫質字體包、有還原漢化，也有試乾淨的0X0000取代後重新漢化

**Reply by GpointChen** [reply]
用高畫質字體包前有先更改設定嗎？

https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94

**Reply by tomokiliyo** [reply]
不知道什麼時候動到了，之前更新這個也沒變的說，改高畫質設定後轉漢化有成功了，非常感謝您！

**Comment by rorocochi** [comment]
大大您好，一直以來都是您的漢化包愛用者，非常感謝您持續提供漢化工具
這次更新後開啟CSV不會跳出平常的漢化&還原畫面，而是直接開啟設置畫面
路徑確定跟以往點選的無誤 但卻會跳出這個提示框
![image_2024-03-21_13-31-51](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/122761195/10bd3154-99a1-40b5-a1c7-eb07ba9743a1)
想了解是否是我哪裡設定跑掉了嗎 謝謝您

**Reply by GpointChen** [reply]
請確認是否使用最新版，並確認該資料夾下有`game/ffxiv_dx11.exe`。

**Reply by rorocochi** [reply]
確定是安裝了最新版，資料夾下也有game/ffxiv_dx11.exe
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/122761195/55eea14e-4d55-4f42-8668-f15b34ce57ef)

**Reply by GpointChen** [reply]
還是你貼一下 `conf/global.properties` 的檔案？可以用記事本打開。

**Reply by rorocochi** [reply]
好的，謝謝麻煩你了~
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/122761195/6048596c-010e-404e-a62e-e5a009cc7e91)

**Reply by GpointChen** [reply]
看起來很正常

或者你試試看將ffxiv_dx11.exe改名成ffxiv.exe，試試看能不能漢化？漢化成功後可以再改回去。

**Reply by rorocochi** [reply]
照著改名的方法試過後 成功了~
真的很謝謝你

**Reply by GpointChen** [reply]
那應該是沒有正確下載到最新版，可以重新下載檢查一下

dx11是這次改版後，漢化器才更新成看ffxiv_dx11.exe

**Reply by rorocochi** [reply]
重新下載後不用更改檔名就成功漢化了，原來是我沒有成功更新成最新版本
謝謝大大不厭其煩幫我查看問題 真的很謝謝您!

**Comment by TangXin0727** [comment]
大大您好, 感謝您一直以來的即時更新.
這次使用漢化器出現了漢化錯誤, 有做過以下嘗試:
1.還原再重新漢化
2.更換原始語言
3.不更換字體
結果均相同. 想請問是否有人遇到類似情況, 我該如何解決? 
感謝您.
![螢幕擷取畫面 2024-03-28 220417](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/52477506/65faaa73-c226-4fc1-8d5c-7a46619bec91)

[debug.log](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/14791039/debug.log)

**Reply by GpointChen** [reply]
試過用backup-20240319.rar還原檔案嗎？

**Reply by TangXin0727** [reply]
昨天有試過不行, 今天有重新下載進行測試, 能成功完成漢化了!!! 遊戲也能正常開啟, 感謝您.

**Comment by bracesc93up** [comment]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/165383159/3ba407bd-3b65-4479-9064-9396d9778735)
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/165383159/1f3855dd-d086-42de-b8bd-dc54f1a60741)
您好 我用兩種方法 以及上面其他人遇到問題的所有方法試過一遍 程式也說漢化完畢 但是進入遊戲還是日文 我也不確定問題出在哪?

**Reply by GpointChen** [reply]
有log嗎？

**Reply by bracesc93up** [reply]
要從哪裡看

**Reply by GpointChen** [reply]
debug.log，漢化器資料夾內

**Reply by bracesc93up** [reply]
[debug.log](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/14810328/debug.log)

**Reply by GpointChen** [reply]
log看起來很正常

你是說你用覆蓋檔案的方式開遊戲也是日文嗎？是全部都是日文還是部份？

另外，是不是試玩版或steam版？

**Reply by bracesc93up** [reply]
1.是的
2.全都是日文
3.不是 是官網買的國際服正式版
上個版本用覆蓋檔案的方式進行漢化有成功 這次也是先用覆蓋檔案但沒成功 才下載漢化工具 但也沒有成功

**Reply by GpointChen** [reply]
覆蓋會失敗比較奇怪，幫我確認一下用覆蓋檔案的方式後檔案是不是長這樣

![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/34565875/b0ae5ab7-9432-4f2d-94bb-16c7a453e5b9)

![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/34565875/48cca8ba-6fc8-488c-9b42-173b62f844b6)

**Reply by bracesc93up** [reply]
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/165383159/441a95e9-a8ce-4b63-a7a8-a2bdd58c728f)

**Reply by GpointChen** [reply]
然後這樣進遊戲還是日文？🤔

你是用官方啟動器還是其他plugin？

**Reply by bracesc93up** [reply]
官方啟動器 我錄製我的螢幕上傳給你看

**Reply by bracesc93up** [reply]
https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/165383159/b22d4c30-124f-48ef-ae61-e28874df595e

**Reply by GpointChen** [reply]
確認一下，是覆蓋完才開遊戲的嗎？

**Reply by bracesc93up** [reply]
是的 原本是錄製覆蓋完再啟動遊戲但是檔案太大丟不上來

**Reply by GpointChen** [reply]
抱歉，看不出原因。
目前的覆蓋檔也有覆蓋英文，或許可以測試一下英文有沒有被替換？

**Reply by Souma-Sumire** [reply]
你的電腦裡是不是存有兩份遊戲呢？你覆蓋的和你實際執行的並不是一份？
我們的檔案結構（修改時間、大小）很明顯不一樣。
![image](https://github.com/GpointChen/FFXIVChnTextPatch-GP/assets/33572696/a7323db9-e3dc-4817-a976-6170526ee3f9)

**Reply by GpointChen** [reply]
看起來有可能？

**Reply by bracesc93up** [reply]
感覺有可能 那我解除安裝重載看看

**Reply by GpointChen** [reply]
如果你是從官方啟動器捷徑開啟遊戲的話，可以試試右鍵>檢視檔案位置看看實際上的檔案位於哪裡

**Reply by bracesc93up** [reply]
重新安裝遊戲之後用覆蓋檔案的方式漢化成功了！ 目前沒有別的問題 感謝大大 辛苦了

**Comment by StevevRYF** [comment]
您好，感谢汉化，请问汉化完的界面全是=是什么原因呢

**Reply by GpointChen** [reply]
有截圖嗎？

=通常代表有字體包不包含的字元

**Comment by StevevRYF** [comment]


**Reply by StevevRYF** [reply]
能发截图么。。

**Reply by GpointChen** [reply]
預設的字體包沒有大廳字體，所以是正常現象。有需要可以參考release頁面另外下載高畫質字體包。

**Reply by StevevRYF** [reply]
哦哦多谢，是不是不能发截图啊我看圈子里有人说，要不我给删了xd

**Reply by GpointChen** [reply]
我個人無所謂，你隨意

**Reply by StevevRYF** [reply]
多谢

[a comment by an automated account is not shown]

**Comment by Lumias02** [comment]
麻煩大大，小弟跪求更新漢化，今天突然使用不了，裝上漢化無法進入游戲
